lib/views/help/books.html.erb

```diff
@@ -0,0 +1,87 @@
+<% @title = 'Access to Information Books' %>
+
+<%= render partial: 'sidebar' %>
+
+<div id="left_column_flip" class="left_column_flip">
+
+  <div id="help_books">
+    <div class="box-out">
+      <div class="box-in">
+        <h1>Access to Information Books</h1>
+        <p>Our list of recommended books about Freedom of Information and how to use it.</p>
```

Comment:
```suggestion
        <p>Our list of recommended books about Freedom of Information and how to use it.</p>
        <p>
          Many of these books will be available, for free, in public libraries.
          <br>
          Not sure where your nearest library is? You can use the 
          <a href="https://libraryon.org/libraries">LibraryOn</a> website to find services close to you.
        </p>
```

Reply:
Nice suggestion – updated with a version of this.